test/controller.test.ts

```diff
@@ -1601,16 +1606,16 @@ contract('Controller', ([owner, accountOwner1, accountOwner2, accountOperator1,
         const vaultCounter = new BigNumber(await controllerProxy.getAccountVaultCounter(accountOwner1))
         assert.isAbove(vaultCounter.toNumber(), 0, 'Account owner have no vault')
 
-        const collateralToDeposit = new BigNumber(await shortOtoken.strikePrice()).dividedBy(1e18)
-        const amountToMint = new BigNumber('1')
+        const collateralToDeposit = new BigNumber(await shortOtoken.strikePrice()).dividedBy(1e12)
+        const amountToMint = new BigNumber(1e18)
```

Comment:
instead of hardcoding these 1e18, 1e12, can you have it calculate it based on the token's decimals? Eg if collateral is USDC, then use usdc.decimals?